src/main/java/com/epam/lowcost/model/Flight.java

```diff
@@ -0,0 +1,80 @@
+package com.epam.lowcost.model;
+
+import java.math.BigInteger;
+import java.util.Date;
+import java.util.Objects;
+
+public class Flight {
+    private int id;
+    private BigInteger initiatedPrice;
```

Comment:
What the reason for using this type? Long is not appropriate?

Reply:
Changed to long